src/main/java/com/flab/bbt/common/CommonResponse.java

```diff
@@ -15,6 +15,26 @@ public CommonResponse(boolean isSuccess, T data, Long errorCode, String message)
         this.message = message;
     }
 
+    public boolean isSuccess() {
+        return isSuccess;
+    }
+
+    public T getData() {
+        return data;
+    }
+
+    public Long getErrorCode() {
+        return errorCode;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public static <T> CommonResponse success(){
+        return new CommonResponse(true, null, null, null);
+    }
+
     public static <T> CommonResponse success(T o){
         return new CommonResponse(true, o, null, null);
```

Comment:
간단한 기능이지만 변수명은 알 수 있게 적는 습관을 가져봅시다.

```suggestion
    public static <T> CommonResponse success(T data){
        return new CommonResponse(true, data, null, null);
```

Reply:
수정해두었습니다.